lib/util.js

```diff
@@ -344,6 +344,14 @@ exports.url =
       }
 
       return url;
+    },
+
+    appendRetryParam: function (option) {
+      let retryUrl = this.appendParam(option.url, 'retryCount', option.retryCount);
+      if (option.includeRetryContext) {
```

Comment:
is it reason related only and that how should it be named, not context?

Reply:
No, I was confused with the query context cache.